scripts/download-openpnp-capture.sh

```diff
@@ -12,13 +12,15 @@ mkdir -p openpnp-capture
 curl -L -o openpnp-capture/source.tar.gz https://github.com/openpnp/openpnp-capture/archive/$1.tar.gz
 tar -C openpnp-capture -xzf openpnp-capture/source.tar.gz --strip 1
 
-mkdir -p openpnp-capture/binaries/darwin
+mkdir -p openpnp-capture/binaries/darwin-x86-64
+mkdir -p openpnp-capture/binaries/darwin-aarch64
 mkdir -p openpnp-capture/binaries/win32-x86
 mkdir -p openpnp-capture/binaries/win32-x86-64
 mkdir -p openpnp-capture/binaries/linux-x86-64
 mkdir -p openpnp-capture/binaries/linux-arm64
 
-curl -L -o openpnp-capture/binaries/darwin/libopenpnp-capture.dylib https://github.com/openpnp/openpnp-capture/releases/download/$1/libopenpnp-capture.dylib
+curl -L -o openpnp-capture/binaries/darwin-x86-64/libopenpnp-capture.dylib https://github.com/openpnp/openpnp-capture/releases/download/$1/libopenpnp-capture.dylib
```

Comment:
Isn't this just downloading the same binary twice? Is it a fat binary with both archs in it?

Reply:
Yep, those were the "unintended issues" that needed a bit of review and that I anticipated happening, will fix, thx!